csharp/Ice/Bidir/slice/AlarmClock.ice

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) ZeroC, Inc.
+
+#pragma once
+
+module EarlyRiser
+{
+    /// Indicates the button pressed when the alarm rang.
+    enum ButtonPressed { Snooze, Stop }
+
+    /// Represents a simple alarm clock. It's the "callback" in this demo, and it's implemented by the client.
```

Comment:
I don't see why we highlight the word "callback" in this sentence, since this is not in the "callback" demo.

Reply:
It's a copy and paste from the Callback demo, and I'd rather keep them in sync.